src/index.html

```diff
@@ -17,16 +17,35 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="wrap">
+      <form
+        class="form big--form"
+        action="#"
+        method="get"
+        data-qa="big"
+      >
+        <div class="icon big--icon"></div>
```

Comment:
Use semantic tags where possible. Instead of `div`, consider using `section` for the wrapper and `header` or `article` for the form containers.